### ISSUE TITLE
Problems with gh-pages

### ISSUE THREAD

**Issue by plumps** [styx-static/styx#28]
> system: macOS Sierra
> nix: 1.11.9
> nixpkgs: master

Hi there, 

using `--gh-pages` doesn't create the rendered version of the page in /gh-pages

Here is what I did:

```diff
➜  plumps-gh git:(master)  styx deploy --init-gh-pages
Cloning into './gh-pages'...
done.
Switched to a new branch 'gh-pages'
rm '.gitignore'
rm 'conf.nix'
rm 'readme.md'
rm 'site.nix'
[gh-pages (root-commit) 59ff43b] initialized gh-pages branch
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 .styx
Successfully created the 'gh-pages' branch.
You can now update the 'gh-pages' branch by running 'styx deploy --gh-pages'.
➜  plumps-gh git:(master) ✗  styx deploy --gh-pages
Building the site
Already on 'gh-pages'
rm '.styx'
-cp: cannot stat '/nix/store/hbbs0gcaaq41v34dz3al2f3w9nahzsvc-styx-site/*': No such file or directory
[gh-pages b5c962a] Styx update - 0dddad6
 1 file changed, 0 insertions(+), 0 deletions(-)
 delete mode 100644 .styx
Successfully updated the gh-pages branch in the 'gh-pages' folder.
Push the 'gh-pages' branch in the 'gh-pages' folder to the GitHub repository to publish your site.
(cd gh-pages && git push -u origin gh-pages) && git push -u origin gh-pages
➜  plumps-gh git:(master) ✗  ls -la gh-pages
total 0
drwxr-xr-x  3 plumps 102 May 25 22:18 .
drwxr-xr-x 10 plumps 340 May 25 22:18 ..
drwxr-xr-x 14 plumps 476 May 25 22:18 .git
```

**Comment by ericsagnes** [comment]
Added a [commit](4ade2c63e4ee2554a213bdb460f0487c36c69c2c) that should help with this.

**Comment by plumps** [comment]
Didn't work for me. 

What I did:

```diff
➜  plumps-gh git:(master) ✗  nix-env -i $(nix-build https://github.com/styx-static/styx/archive/master.tar.gz)
➜  plumps-gh git:(master) ✗  styx --help
Styx 0.7.0-dev -- Styx is a functional static site generator in Nix expression language.
...
➜  plumps-gh git:(master) ✗  rm -rf ./gh-pages
➜  plumps-gh git:(master) ✗  styx deploy --init-gh-pages
Cloning into './gh-pages'...
done.
Switched to a new branch 'gh-pages'
rm '.gitignore'
rm 'conf.nix'
rm 'readme.md'
rm 'site.nix'
[gh-pages (root-commit) f97ec5a] initialized gh-pages branch
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 .styx
Successfully created the 'gh-pages' branch.
You can now update the 'gh-pages' branch by running 'styx deploy --gh-pages'.
➜  plumps-gh git:(master) ✗  styx deploy --gh-pages
Building the site
Already on 'gh-pages'
rm '.styx'
-cp: cannot stat '/nix/store/hbbs0gcaaq41v34dz3al2f3w9nahzsvc-styx-site/*': No such file or directory
[gh-pages 2299853] Styx update - 0dddad6
 1 file changed, 0 insertions(+), 0 deletions(-)
 delete mode 100644 .styx
Successfully updated the gh-pages branch in the 'gh-pages' folder.
Push the 'gh-pages' branch in the 'gh-pages' folder to the GitHub repository to publish your site.
(cd gh-pages && git push -u origin gh-pages) && git push -u origin gh-pages
```

**Comment by ericsagnes** [comment]
It seems that the build produced no files, there might be a problem in `site.nix`.  
Is `/nix/store/hbbs0gcaaq41v34dz3al2f3w9nahzsvc-styx-site/` empty?

**Comment by ericsagnes** [comment]
Should be fixed in 0.7.0.
Please reopen the issue if the problem happens again.
